generator/geo_objects/key_value_storage.hpp

```diff
@@ -0,0 +1,36 @@
+#pragma once
+
+#include "3party/jansson/myjansson.hpp"
```

Comment:
docs/CPP_STYLE.md:

`
Order of directories in includes: "current_dir/current_file.hpp", includes from other dirs sorted by dependencies (e.g. indexer, then coding,     then base), "defines.hpp", C++ standard library headers, boost headers, 3party.
`

Reply:
done